src/index.html

```diff
@@ -1,13 +1,433 @@
+    <link rel="stylesheet" href="styles/style.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page_body">
+    <header class="header">
+      <div class="container">
+        <div>
+          <div class="top-bar">
+            <a class="top-bar__logo-link" href="#">
+              <img class="top-bar__logo" src="./images/logo-2x.png"
+              alt="The Met Logo">
+            </a>
+
+            <div class=" page_phone" id="phone">
+              <a href="tel:+1 212-923-3700" class="menu_contacts-text">The Met Cloisters<br>+1 212-923-3700</a>
+
+              <a href="tel:+1 212-535-7710" class="menu_contacts-text">The Met Fifth Avenue <br>+1 212-535-7710</a>
+            </div>
+
+            <div class="top-bar__icons">
+              <a href="#phone" class="icon icon--phone">
+              </a>
+
+              <a href="#menu" class="icon icon--menu">
+              </a>
+            </div>
+          </div>
+          <div class="header__middle">
+            <h1 class="header__title">Welcome
+              <br>
+                to The MET
+            </h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="menu page_menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu">
+            <div class="top-bar  menu__top ">
+              <a class="top-bar__logo-link" href="#">
+                <img class="top-bar__logo" src="./images/logo-2x.png"
+                alt="The Met Logo">
+              </a>
+
+              <div class="top-bar__icons">
+                <a href="#" class="icon icon--close ">
+                </a>
+              </div>
+            </div>
+
+            <div class="menu_bottom">
+              <nav class="nav menu__nav">
+                <ul class="nav__list">
+                  <li class="nav__item">
+                    <a class="nav__link" href="#home">HOME</a>
+                  </li>
+
+                  <li class="nav__item">
+                    <a class="nav__link" href="#about-us">ABOUT US</a>
+                  </li>
+
+                  <li class="nav__item">
+                    <a class="nav__link" href="#exhibitions">NOW ON VIEW</a>
+                  </li>
+
+                  <li class="nav__item">
+                    <a class="nav__link" href="#explore-online">EXPLORE ONLINE</a>
+                  </li>
+
+                  <li class="nav__item">
+                    <a class="nav__link" href="#contacts">CONTACTS</a>
+                  </li>
+                </ul>
+              </nav>
+
+              <div class="menu_contacts">
+                <div><a href="tel:+1 212-535-7710" class="menu_contacts-text menu_contacts--link"><div></div>The Met Fifth Avenue <br>+1 212-535-7710</a></div>
```

Comment:
Need to fix format phone everywhere only ib href
```suggestion
                <div><a href="tel:+12125357710" class="menu_contacts-text menu_contacts--link"><div></div>The Met Fifth Avenue <br>+1 212-535-7710</a></div>
```